src/Microsoft.Health.Dicom.SqlServer/Features/Retrieve/SqlInstanceStoreV1.cs

```diff
@@ -109,15 +111,16 @@ public SqlInstanceStoreV1(SqlConnectionWrapperFactory sqlConnectionWrapperFactor
                             rStudyInstanceUid,
                             rSeriesInstanceUid,
                             rSopInstanceUid,
-                            watermark));
+                            watermark,
+                            partitionEntry));
```

Comment:
At V1, we don't have a concept of partition, shouldn't it accept default value?

Reply:
We're talking about v1 of the SQL schema, right? We don't necessarily care about about the SQL schema here do we? As the partition is being read/default by the caller?